src/inferencer/inferencer.ts

```diff
@@ -573,11 +606,14 @@ function infer(statement: es.Node, environmentToExtend: Map<any, any> = emptyMap
       inferConditionals(statement)
       return
     }
-    case 'FunctionDeclaration': {
+    case 'FunctionDeclaration':
```

Comment:
Why is the Function Declaration case blank?

Reply:
Because it's the same case as the one directly below i.e. `case 'ArrowFunctionExpression':` - this case thing stacks

Reply:
Noted